data/wanderer/wanderers middle.txt

```diff
@@ -20,6 +20,319 @@ mission "Wanderers: Pre-Colony"
+			`	"[Truly? Honestly?]" he asks. When you nod your head he hands you a data chip and says, "Take this to Vara K'chrai and they will make the arrangements. We need construction equipment, materials and workers and I can't even request them directly since communications don't work through the wormhole.`
+			`	You turn to go as group of Wanderers with surveying equipment walk up. They are all dressed in hard hats and safety gear. The construction supervisor you've been speaking to pulls up a hologram of the site to discuss with the surveyors. As you walk off, the translator just picks up. "The Eye opens and we are getting help from the [alien, stranger] and not our own flock."`
+				accept
+
+	on complete
+		conversation
+			`As you land at the spaceport, you're sure you see a few of the ships you saw on <planet> with various outfits stacked for installation. It's hard to find anyone who speaks Hai, but you find your way to the main spaceport office and hand over the chip. The person at the desk chitters excitedly and calls someone over. They quickly enter information into their systems and then hand the chip back to you. After a few moments of confusion where you end up making your Hai ear gesture, they call up an app and speak into it. Your translator picks up "Job board. Job board." Hopefully everything has been booked for you.`
+
+mission "Wanderers: Spera Anatrusk Outfitter 2a"
+	landing
+	name "Construction workers to <planet>"
+	description "Now that the harvest is over, bring workers from <stopovers> to <destination> to build the new outfitter. Payment is <payment>."
+	blocked "The building supervisor at <destination> has [requisitioned, hired] more workers than you can carry. You will need <capacity> to transport these workers."
+	source "Vara K'chrai"
+	destination "Spera Anatrusk"
+	stopover "Vara Rakak"
+	passengers 75
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 1: offered"
+
+	on stopover
+		dialog "The construction workers board your ship chattering excitedly about their chance to pass through the Eye to new worlds."
+
+	on visit
+		dialog "You've arrived at <planet> but the escort with the construction workers hasn't landed. You realize you need to depart and wait for them to arrive."
+
+	on complete
+		dialog "The construction supervisor greets the <ship> as you land at the spaceport site. He hands you a credit chip for <payment> and immediately begins assigning work to the teams."
+		payment 750000
+
+mission "Wanderers: Spera Anatrusk Outfitter 2b"
+	landing
+	name "Building materials to <planet>"
+	description "Bring <cargo> from <stopovers> to <destination> to build a new outfitter. Payment is <payment>."
+	blocked "The building supervisor at <destination> has [requisitioned, purchased] more building materials than you can carry. You will need <capacity> to transport it."
+	source "Vara K'chrai"
+	destination "Spera Anatrusk"
+	stopover "Varu Ek'lak'lai"
+	cargo "prefabricated building materials" 500
```

Comment:
That's WAY too much cargo for a campaign mission; this and the next one.

If you want there to be a large amount of passengers and cargo transported then these should be escort missions with the necessary cargo and passengers on the escorts instead of giving such large objectives to the player.

Reply:
Is there a rule of thumb for what this break point should be? This is half the capacity of Deep River. If it's based on a single ship model, it could range from 40/80 tons for some human heavy warships to 500 for a Bactrian. I'm not arguing the point, I'm looking for guidance so I write to that spec in the future.

Reply:
I've cut this down to 200 for this job and 185 for the next one. This is based on a check of transport jobs in Wanderer space. The largest in my short sample was 200. I would still appreciate guidance on what you expect to see.